package.json

```diff
@@ -28,30 +28,29 @@
   "homepage": "https://mobxjs.github.io/mobx",
   "peerDependencies": {
     "mobx": "^2.6.3 || ^3.0.0",
-    "react": "^0.13.0 || ^0.14.0 || ^15.0.0"
+    "react": "^0.13.0 || ^0.14.0 || ^15.0.0 || ^16.0.0-0"
   },
   "devDependencies": {
-    "babel-core": "^6.11.4",
-    "babel-loader": "^6.2.4",
-    "babel-plugin-transform-class-properties": "^6.11.5",
+    "babel-core": "^6.24.1",
+    "babel-loader": "^6.4.1",
+    "babel-plugin-transform-class-properties": "^6.24.1",
     "babel-plugin-transform-decorators-legacy": "^1.3.4",
-    "babel-preset-es2015": "^6.9.0",
-    "babel-preset-react": "^6.16.0",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-react": "^6.24.1",
     "babelify": "^7.3.0",
-    "browserify": "^13.1.0",
+    "browserify": "^14.3.0",
     "empty-module": "0.0.2",
-    "enzyme": "^2.6.0",
-    "jquery": "^2.1.4",
-    "lodash": "^4.0.1",
-    "mobx": "^2.7.0 || ^3.0.0",
-    "react": "^15.4.2",
-    "react-addons-test-utils": "^15.4.2",
-    "react-dom": "^15.4.2",
+    "enzyme": "^2.8.0",
+    "lodash": "^4.17.4",
+    "mobx": "^3.1.9",
+    "react": "^15.5.3",
+    "react-addons-test-utils": "^15.5.1",
+    "react-dom": "^15.5.3",
     "tap-spec": "^4.1.1",
-    "tape": "^4.2.2",
-    "tape-run": "2.1.0",
-    "typescript": "^2.1.4",
-    "webpack": "^1.13.1"
+    "tape": "^4.6.3",
+    "tape-run": "3.0.0",
```

Comment:
Ok, this commit seems to fix the travis issue. @andykog  can you downgrade `tape-run` in this branch?